packages/pricing/src/migrations/Migration20231101232834.ts

```diff
@@ -35,10 +35,7 @@ export class Migration20231101232834 extends Migration {
         SET title = name
     `)
 
-    this.addSql(`alter table "price_list"
-        alter column "title" set not null `)
-
-    this.addSql('alter table "price_list" drop column if exists "name";')
```

Comment:
we're removing this as the migration scripts require the column name to exist after the module migrations are done. Quite the annoying chicken and egg situation, will come up with a better solution after, but this should fix the bug for all cases of isolated and shared.

Reply:
Not sure I understand. Can I get you to expand on this?

Reply:
to migrate the data from core to pricing compatible schema, we need to run the following:

1. Run migrations for core
2. Run migrations for pricing module
3. Run data migration to move from core to pricing module

on the 2nd run, we remove drop the column name. To migrate the data (step 3), we use typeorm, which recognizes only name as a column, but when we fetch the data from typeorm, it throws an error because we already dropped it in Step 2.

Reply:
Got it, thanks. I guess we will need to include a separate migration that needs to be run after then. Don't you think it is best to do this now, considering that we will already have to prepare an upgrade guide? That way, we don't have to think about it later on.